src/ops/kernels/element_binary_kernels.cu

```diff
@@ -237,6 +267,24 @@ __global__ void elewise_binary_backward_kernel(coord_t volume,
                       beta * in2_grad[i];
         break;
       }
+      case OP_EW_MAX: {
+        in1_grad[i] = (in1[i] >= in2[i])
+                          ? alpha * out_grad[i] + beta * in1_grad[i]
+                          : beta * in1_grad[i];
+        in2_grad[i] = (in2[i] >= in1[i])
+                          ? alpha * out_grad[i] + beta * in2_grad[i]
+                          : beta * in2_grad[i];
+        break;
+      }
+      case OP_EW_MIN: {
+        in1_grad[i] = (in1[i] <= in2[i])
+                          ? alpha * out_grad[i] + beta * in1_grad[i]
+                          : beta * in1_grad[i];
+        in2_grad[i] = (in2[i] <= in1[i])
+                          ? alpha * out_grad[i] + beta * in2_grad[i]
+                          : beta * in2_grad[i];
+        break;
+      }
```

Comment:
@jiazhihao @lockshaw, do the gradients look fine? Specifically, for the case when the inputs are equal, I am allowing the gradients to propagate to both inputs.
Also, are these actually being used somewhere? It appears to me that the CUDNN implementations below are the ones that are actually being used.

Reply:
Might be a good idea to test against pytorch, a la #608?

Reply:
@xinhaoc Xinhao, can you also include min and max tests in PyTorch alignment?

Reply:
Yes I will do that